libs/platform-configuration-impl/src/main/java/org/codice/ddf/configuration/impl/ConfigurationWatcherImpl.java

```diff
@@ -33,40 +33,40 @@ public ConfigurationWatcherImpl() {}
   /**
    * Helper method to get the hostname or IP from the configuration
    *
-   * @return the value associated with {@link SystemBaseUrl#HOST} property name
+   * @return the value associated with {@link SystemBaseUrl#INTERNAL_HOST} property name
```

Comment:
❗️ The javadocs in this file are `INTERNAL` but the return values are `EXTERNAL`.